src/Illuminate/Database/Query/Builder.php

```diff
@@ -922,17 +922,18 @@ public function orWhereRaw($sql, $bindings = [])
     {
         return $this->whereRaw($sql, $bindings, 'or');
     }
-
+    
+    
     /**
      * Add a "where in" clause to the query.
      *
-     * @param  string  $column
+     * @param  string|array  $column
      * @param  mixed  $values
      * @param  string  $boolean
      * @param  bool  $not
      * @return $this
      */
-    public function whereIn($column, $values, $boolean = 'and', $not = false)
+    public function whereIn($columns, $values, $boolean = 'and', $not = false)
```

Comment:
This `s` broke it.

Reply:
Yeh Thanks, I thought about whether or not I should rename the $argument but decided in the end not to, just forgot to change it back.